client/notices/index.js

```diff
@@ -11,7 +11,7 @@ var list = { containerNames: {} };
 Emitter( list );
 var delayedNotices = [];
 
-module.exports = {
+const notices = module.exports = {
```

Comment:
This looks a bit awkward -- you might consider doing it in two steps (define the const first, export it later).